package.json

```diff
@@ -20,6 +20,7 @@
     "color-blend": "^4.0.0",
     "crypto-browserify": "^3.12.0",
     "date-fns": "^2.29.3",
+    "history": "^5.3.0",
```

Comment:
what does the issue solve this library?

Reply:
since react router v6 doesn't support for preventing transitions and navigator (aka history) object has no longer block method exposed [it looks like now it requires using a custom history object and HistoryRouter](https://stackoverflow.com/questions/74064963/react-router-dom-v6-prevent-transition-without-prompt)